components/Forms/ChangePasswordForm/ChangePasswordForm.js

```diff
@@ -1,16 +1,15 @@
 import React, { useState } from 'react';
 import { string, func, shape } from 'prop-types';
-import { Field, Formik } from 'formik';
+import { Field, Formik, Form } from 'formik';
 import * as Yup from 'yup';
-import Button from 'components/Buttons/Button/Button';
-import Form from 'components/Form/Form';
-import Input from 'components/Form/Input/Input';
-import Alert from 'components/Alert/Alert';
-import { CHANGE_PASSWORD_FORM_ERROR } from 'common/constants/testIDs';
-import { validationErrorMessages } from 'common/constants/messages';
-import { getServerErrorMessage } from 'common/utils/api-utils';
-import { minimumPasswordLength } from 'common/constants/validations';
-import { hasRequiredCharacters } from 'common/utils/validator-utils';
+import Button from '../../Buttons/Button/Button';
+import Input from '../../Form/Input/Input';
+import Alert from '../../Alert/Alert';
```

Comment:
Can you adjust imports in your PR to leverage our path aliases whenever possible?

For example, `'components/Buttons/Button/Button';` and `'common/constants/testIDs';`

Reply:
All done :-)